src/metaopt/algo/space.py

```diff
@@ -119,6 +118,8 @@ def __init__(self, name, prior, *args, **kwargs):
         # Default shape `None` corresponds to 0-dim (scalar) or shape == ().
         # Read about ``size`` argument in
         # `scipy.stats._distn_infrastructure.rv_generic._argcheck_rvs`
+        if 'size' in kwargs:
+            raise ValueError("Use 'shape' keyword only instead of 'size'.")
```

Comment:
Why add this check for `size`? `kwargs` is built from a call to `Dimension.__init__`, so I would call it more an "argument" than a "keyword".

Reply:
If you asking why checking about a keyword named 'size', that's for UI reasons in case someone writes something like `--var-name~'gaussian(5, 4, size=(3, 4))'` I am rejecting that for the more numpy like use of `--var-name~'gaussian(5, 4, shape=(3, 4))'`. Notice the incoherency in numpy (`shape`) and scipy (which expects `size`).

Otherwise, if you are asking why 'size' in kwargs and not in args, the reason is I would not achieve my above goal.

Reply:
Question was about size vs shape indeed. "argument" vs "keyword" was about the text in the ValueError. That's fine.